### DISCUSSION TITLE
Error in the file trulens_eval/feedback/provider/endpoint/openai.py

### DISCUSSION THREAD

**Discussion by masdeval** [truera/trulens#805] [Q&A] [answered]
Error in line 70:

client: Union[oai.OpenAI, oai.AzureOpenAI] = pydantic.Field(exclude=True)

AttributeError: module 'openai' has no attribute 'OpenAI'

Versions:
trulens                     0.13.4
trulens-eval                0.20.3

Any ideas?

**Comment by joshreini1** [comment, marked as answer]
Hey @masdeval please update to trulens-eval==0.22.0. Let me know if that solves the issue here. If not, please also let me know your python version

**Reply by masdeval** [reply]
It worked. 

Thank you!
